### ISSUE TITLE
TiKV in 5 minutes docs code with Python client does not work for batch get for Raw KV

### ISSUE THREAD

**Issue by karuppiah7890** [tikv/website#286]
## Change Request

https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/ has python code for raw kv, with name `test_raw.py` and this code produces this output -

```bash
$ python3 test_raw.py 
b'Hello'
None
[(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'Raw KV')]
```

Note the `None` for the batch get result, where the code is `print(client.batch_get([b"k1", b"k3"]))`

### Describe the problem

- Relevant URL(s): https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/ , https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/#use-python

### Suggest an improvement/fix

Not sure what the improvement here is, I haven't debugged or deep dived into the problem. If I find anything, I'll post here

**Comment by karuppiah7890** [comment]
For similar code using Transactional KV API, batch get worked and the output was like this -

```bash
$ python3 test_txn.py 
b'Hello'
[(b'k1', b'Hello'), (b'k3', b'World')]
[(b'k1', b'Hello'), (b'k2', b','), (b'k3', b'World'), (b'k4', b'!'), (b'k5', b'TXN KV')]
```

Note how the output is `[(b'k1', b'Hello'), (b'k3', b'World')]` for the batch get

**Comment by marsishandsome** [comment]
@andylokandy PTAL

**Comment by marsishandsome** [comment]
@karuppiah7890 please install tikv-client using `pip3 install tikv-client`.
this bug is fixed in tikv-client v0.0.2

**Comment by karuppiah7890** [comment]
Awesome! Thanks @marsishandsome !
